Dockerfile.s390x

```diff
@@ -48,6 +48,21 @@ RUN apt-get update && apt-get install -y \
 	tar \
 	--no-install-recommends
 
+# install seccomp: the version shipped in jessie is too old
+ENV SECCOMP_VERSION 2.3.0
+RUN set -x \
+	&& export SECCOMP_PATH="$(mktemp -d)" \
+	&& curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
+		| tar -xzC "$SECCOMP_PATH" --strip-components=1 \
+	&& ( \
+		cd "$SECCOMP_PATH" \
+		&& ./configure --prefix=/usr/local \
+		&& make \
+		&& make install \
+		&& ldconfig \
+	) \
+	&& rm -rf "$SECCOMP_PATH"
+
```

Comment:
You're going to want to add seccomp as a DOCKER_BUILDTAGS (line 152) as well. Also you should take a look at https://github.com/seccomp/libseccomp-golang/issues/7, it looks like libseccomp is missing some go-bindings on p,z,and mips. This is a big enough deal that we don't build seccomp yet with P, but your mileage may vary.